pkg/diff/diff.go

```diff
@@ -969,10 +969,10 @@ func CreateTwoWayMergePatch(orig, new, dataStruct interface{}) ([]byte, bool, er
 	return patch, string(patch) != "{}", nil
 }
 
-// HideSecretData replaces secret data values in specified target, live secrets and in last applied configuration of live secret with stars. Also preserves differences between
+// HideSecretData replaces secret data & optional annotations values in specified target, live secrets and in last applied configuration of live secret with stars. Also preserves differences between
 // target, live and last applied config values. E.g. if all three are equal the values would be replaced with same number of stars. If all the are different then number of stars
```

Comment:
This appears to be a lie.

The code uses `+` (plus) characters not `*` (star) characters.

It's a preexisting condition, but it'd be nice if it were fixed (in a distinct commit).

Reply:
Updated the comment to reflect this.